### PR TITLE
Fix merge conflicts with 0.52 branch for async changes

### PR DESCRIPTION
**Pull request by allansson**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/grafana/k6-DefinitelyTyped/pull/50

### CODE REVIEW

File: types/k6/test/browser.ts
Comment (allansson) [inline types/k6/test/browser.ts:1]: This is a huge PR but this is the only thing that was changed by me. Other files are due to the 0.52 pulling in the latest version from upstream.

Most of the conflicts were cause between the `tap` api already being updated to async and the indentation added when wrapping the tests in a function.

**Review by allansson — COMMENTED** [review]
Self-review

**Comment by allansson** [comment]
Apologies to anyone being pinged about signing our CLA. It's my error because of doing a merge commit from master into this branch. You of course don't have to sign anything.

**Comment by james-pre** [comment]
@allansson I've updated my GitHub username from @dr-vortex to @james-pre. Please update your comment(s), and I can delete this one. I'm writing this comment when I can't make the change myself.

**Pull request merged**